src/amdgpu_libdrm.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#define LIBDRM_MAX_DEVICES 32
+
+#define LIBDRM_SAMPLE_DELAY 3500
+#define LIBDRM_SAMPLE_BUF_SIZE 512
+
+#define LIBDRM_GRBM_STATUS 0x8010
+
+enum LIBDRM_GRBM_BITS {
+    LIBDRM_GRBM_BUSY_BIT = 1U << 31
+};
+
+struct libdrm_sample {
+    bool busy_bit;
+};
+
+struct libdrm_stats {
+    int busy;
+};
+
```

Comment:
All the above information is private to amdgpu_libdrm.cpp, so it should be moved there.